pkg/kubernetes/client/v1alpha1/client.go

```diff
@@ -82,6 +83,13 @@ type getKubeMasterIPFunc func(env.ENVKey) string
 //  typed function makes it simple to mock
 type getKubeConfigPathFunc func(env.ENVKey) string
 
+// getKubernetesDynamicClientsetFunc provides the abstraction to get
```

Comment:
spellcheck !! name does not match with type

Reply:
Done